client/activity.coffee

```diff
@@ -264,7 +265,10 @@ bind = ($item, item) ->
           for each in map.sitemap
             sites = unfilteredPages[each.slug]
             unfilteredPages[each.slug] = sites = [] unless sites?
-            sites.push {site: site, page: {slug: each.slug, title: each.title, date: each.date}}
+            if _.findIndex(sites, ['site', site]) is -1
```

Comment:
this could be changed to `unless _.find(sites, ['site', site])` to ease comprehension.